### PR TITLE
Fix dashboard publish link xss

### PR DESCRIPTION
**Pull request by flops**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9145-fix-publish-link-xss-612/component-dialog-switch-public-view-612/en/">component.dialog-switch-public.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9145-fix-publish-link-xss-612/component-dialog-switch-public-view-612/ru/">component.dialog-switch-public.view (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

File: src/ui/components/EntryDialogues/DialogSwitchPublic/DialogPublic/AuthorSection/AuthorSection.tsx [hunk 1]
```diff
@@ -21,29 +23,32 @@ type Props = {
...
-                <TextInput
-                    value={authorData.link}
-                    disabled={disabled}
-                    onUpdate={(value) => onChange({link: value})}
-                ></TextInput>
+
+            <div className={b('form')}>
+                <FormRow label={i18n('label_author-text')} className={b('form-row')}>
+                    <TextInput
+                        error={validationErrors.text ?? false}
```
Comment (imsitnikov) [inline src/ui/components/EntryDialogues/DialogSwitchPublic/DialogPublic/AuthorSection/AuthorSection.tsx:38]: Do we have any validation errors for this field?

**Review by imsitnikov — DISMISSED** [review]

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

File: src/ui/components/EntryDialogues/DialogSwitchPublic/DialogPublic/AuthorSection/AuthorSection.tsx (hunk 1 above)
Comment (flops) [reply src/ui/components/EntryDialogues/DialogSwitchPublic/DialogPublic/AuthorSection/AuthorSection.tsx:38]: Yep, not yet implemented, but i've done validationErrors for both fields in form

**Review by Marginy605 — APPROVED** [review]

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**